modules/exploits/multi/http/phpmailer_arg_injection.rb

```diff
@@ -60,7 +60,10 @@ def initialize(info = {})
       ])
     register_advanced_options(
       [
-        OptInt.new('WAIT_TIMEOUT', [true, 'Seconds to wait to trigger the payload', 300])
+        OptInt.new('WAIT_TIMEOUT', [true, 'Seconds to wait to trigger the payload', 300]),
+        OptString.new('NAME_FIELD', [true, 'Name of the element for the Name field', 'name']),
+        OptString.new('EMAIL_FIELD', [true, 'Name of the element for the Email field', 'email']),
+        OptString.new('MESSAGE_FIELD', [true, 'Name of the element for the Message field', 'message'])
```

Comment:
Added Advanced Options

Reply:
So I know that `WAIT_TIMEOUT` is screaming snake case, but typically advanced options are camel case (e.g. `WaitTimeout`, `NameField`, etc.). Also would you mind adding these options to the [module docs](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/exploit/multi/http/phpmailer_arg_injection.md)?

There should be an options section like this after the Verification Steps and before the Scenarios section:

```
## Options

### NameField
*This is an advanced option.*

Name of the element for the Name field.

...

## Scenarios
```